packages/react/src/Banner/Banner.features.stories.tsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react'
+import {action} from '@storybook/addon-actions'
+import type {Meta} from '@storybook/react'
+import {Banner} from '../Banner'
+import Link from '../Link'
+import {VisuallyHidden} from '../internal/components/VisuallyHidden'
+
+const meta = {
+  title: 'Drafts/Components/Banner/Features',
+  component: Banner,
+} satisfies Meta<typeof Banner>
+
+export default meta
+
+export const Critical = () => {
+  return (
+    <Banner
+      title="Critical"
+      description={
```

Comment:
For my own React learning, I'm curious why you went with introducing `description` as a prop, versus a slot!

Reply:
@khiga8 love this question. Just want to double-check, by slot did you mean the html `<slot>` or more so the PVC slots idea?

I think the tl;dr for this is that this setup is (at least as I understand it) how slots are typically done for React components. I'm definitely thinking of [this article](https://engineering.atspotify.com/2023/05/multiple-layers-of-abstraction-in-design-systems/) when I'm saying that though so I'm not sure how ubiquitous that is. Specifically, they have a good breakdown that I love to keep in mind in the React world:

![A row breakdown of the different types of Component API decisions that you can make. On the left side, there is a config pattern where the component takes in dedicated props and strings that are their values for a Card component. In the middle is the slots pattern that expands the values of props to include other react components. On the far right side, they use a custom pattern where each of the props are now children of the card component](https://github.com/primer/react/assets/3901764/80509df8-e6cb-4703-87a3-d50866230644)

Hope that makes sense, would love to know what you think! 👀